views/agency-requests/view_participant.php

```diff
@@ -17,58 +17,60 @@
 
-$familyResult = pg_fetch_assoc($familyInfo);
 
-//all info related to agency consent
-$agencyInfo = $db->query("	SELECT participants.participantid, people.middleinit, people.firstname, people.lastname, 
-							 participants.dateofbirth, participants.sex, participants.race, 
-							 contactagencymembers.agency, contactagencymembers.phone, contactagencymembers.email, 
-							 agencyreferral.hasagencyconsentform 
-							 FROM participants 
-							 INNER JOIN people ON participants.participantid = people.peopleid 
-							 LEFT JOIN forms ON participants.participantid = forms.formid 
-							 LEFT JOIN agencyreferral ON forms.formid = agencyreferral.agencyreferralid 
-							 LEFT JOIN contactagencyassociatedwithreferred ON agencyreferral.agencyreferralid = contactagencyassociatedwithreferred.agencyreferralid 
-							 LEFT JOIN contactagencymembers ON contactagencyassociatedwithreferred.contactagencyid = contactagencymembers.contactagencyid  
-							 WHERE participants.participantid = $1", [$peopleid]);
-$agencyResult = pg_fetch_assoc($agencyInfo);
+// Grab agency information
+$db->prepare("get-participant-agency-info","SELECT participants.*, agency, phone, email, 
+                            ismaincontact,
+                            hasagencyconsentform, hasmentalhealth,hasspecialneeds, hasmentalhealth,
+                            people.*
+                            FROM participants
+                            INNER JOIN people on participants.participantid = people.peopleid
+                            INNER JOIN forms ON participants.participantid = forms.participantid
+                            LEFT JOIN agencyreferral ON forms.formid = agencyreferral.agencyreferralid
+                            LEFT JOIN contactagencyassociatedwithreferred ON agencyreferral.agencyreferralid = contactagencyassociatedwithreferred.agencyreferralid
+                            LEFT JOIN contactagencymembers ON contactagencyassociatedwithreferred.contactagencyid = contactagencymembers.contactagencyid
+							 WHERE participants.participantid = $1");
+$result = $db->execute("get-participant-agency-info",[$peopleid]);
+$agencyResult = pg_fetch_assoc($result);
+extract($agencyResult);
 
-//grab address information
+// Grab address information
 $db->prepare("get-participant-addresses", "	SELECT * 
 											FROM participants
 											INNER JOIN people ON participants.participantid = people.peopleid
 											LEFT JOIN forms as fo ON participants.participantid = fo.participantid
 											LEFT JOIN addresses ON  fo.addressid = addresses.addressid
 											LEFT JOIN zipcodes ON zipcodes.zipcode =addresses.zipcode
-											WHERE participants.participantid = $1");
+											WHERE participants.participantid =  $1 ORDER BY employeesigneddate DESC LIMIT 1");
 $result = $db->execute("get-participant-addresses",[$peopleid]);
 $address = pg_fetch_assoc($result);
 extract($address);
 $addressid = $address['addressid'];
```

Comment:
I think I've mentioned this before--you can just access `$addressid` without doing this assignment since you're doing `extract()` on the `$address` associative array.

Reply:
You did! Changed it